var/spack/repos/builtin/packages/pacparser/package.py

```diff
@@ -14,16 +14,19 @@ class Pacparser(MakefilePackage):
 
     version('1.3.7', sha256='eb48ec2fc202d12a4b882133048c7590329849f32c2285bc4dbe418f29aad249')
 
+    # possible race condition in makefile
+    parallel = False
+
     depends_on('python', when='+python')
     depends_on('py-setuptools', when='+python', type=('build', 'run'))
 
     variant('python', default=False,
             description='Build and install python bindings')
 
     def build(self, spec, prefix):
-        make('-C', 'src')
+        make('CC=cc', 'CCC=c++', '-C', 'src')
```

Comment:
I think this is the Spack way of explicitly setting compiler. @tldahlgren correct me if I'm wroing.

```suggestion
        make('CC="%s"' % self.compiler.cc, 'CXX="%s"' % self.compiler.cxx, '-C', 'src')
```